pkg/diff/schema_migration_plan_test.go

```diff
@@ -42,150 +42,6 @@ var (
 	}
 
 	schemaMigrationPlanTestCases = []schemaMigrationPlanTestCase{
-		{
-			name: "No-op",
```

Comment:
What's the purpose of removing these test cases?

Reply:
The schema migration plan cases test "how" the migration is done, not just the migration's corectness. The latter are tested by the acceptance tests.

Since there are many "no-op" acceptance tests that assert the generated plan is empty, we don't need this test case. I'm trying to minimize the number of these test cases, since it's tricky to maintain the before and after schemas.